LEGO1/lego/legoomni/include/pizza.h

```diff
@@ -31,13 +31,20 @@ class Pizza : public IsleActor {
 	undefined4 HandleEndAction(MxEndActionNotificationParam&) override; // vtable+0x74
 	undefined4 VTable0x80(MxParam&) override;                           // vtable+0x80
 
+	void FUN_100382b0();
+	void FUN_10038380();
+
 	// SYNTHETIC: LEGO1 0x100380e0
 	// Pizza::`scalar deleting destructor'
 
 private:
 	undefined4 m_unk0x7c; // 0x7c
 	undefined4 m_unk0x80; // 0x80
-	undefined4 m_unk0x84; // 0x84
+
+public:
+	undefined* m_unk0x84; // 0x84
```

Comment:
Same thing here, generally inline getter functions should be used. Look at the other classes such as `legocontrolmanager.h` above to see some examples.